### ISSUE TITLE
Standardize filepath column name across tables

### ISSUE THREAD

**Issue by leilani-reich** [sailuh/kaiaulu#173]
There is some inconsistency in the naming of parameters to specify a file path in parser.R.

In regards to https://github.com/sailuh/kaiaulu/issues/168#issuecomment-1483783147, I've checked the parser.R file functions and parse_gitlog() calls the file column "file_pathname" https://github.com/sailuh/kaiaulu/blob/716a46e546d5293f3b2856cace62311c95aae977/R/parser.R#L80

However, I also saw the file column called "filename" in parse_git_blame() https://github.com/sailuh/kaiaulu/blob/716a46e546d5293f3b2856cace62311c95aae977/R/parser.R#L174

and "filepath" in parse_line_type_file() https://github.com/sailuh/kaiaulu/blob/716a46e546d5293f3b2856cace62311c95aae977/R/parser.R#L717.

Also, should this be in the project board and/or a specific milestone? Thanks

**Comment by carlosparadis** [comment]
> Also, should this be on the project board and/or a specific milestone? Thanks

No, since this is not a deliverable I expect from your group, and there are enough tasks as is. You can leave without milestone. I will get around working the consistency of table names in the future. There may be other columns like this. Thank you for asking 👍
